ivy/functional/frontends/tensorflow/func_wrapper.py

```diff
@@ -92,3 +92,35 @@ def new_fn(*args, **kwargs):
 
 def to_ivy_arrays_and_back(fn: Callable) -> Callable:
     return outputs_to_tensorflow_array(inputs_to_ivy_arrays(fn))
+
+
+def map_raw_ops_alias(alias: callable, **function_kwargs):
+    """
+    Mapping the raw_ops function with its respective frontend alias function,
+    as the implementations of raw_ops is way similar to that of frontend functions,
+    except that only arguments are passed as key-word only in raw_ops functions.
+
+    Parameters
+    ----------
+    alias:
+        The frontend function that is being referenced to as an alias to the
+        current raw_ops function.
+    functions_kwargs:
+        All inputs to the raw_ops as keyword only arguments.
+
+    Returns
+    -------
+    ret
+        The output of the wrapped function passed key-word only arguments as input.
+    """
+
+    def _wrap_raw_ops_alias(fn: callable) -> callable:
+        @functools.wraps(fn)
+        def _wraped_fn(*args, **kwargs):
+            kwargs.update(zip(fn.__code__.co_varnames, args))
+            return fn(**kwargs)
+
+        return _wraped_fn
+
+    wrapped_function = _wrap_raw_ops_alias(alias)
+    return wrapped_function(**function_kwargs)
```

Comment:
We shouldn't be calling the function, we should be returning it instead, so it should be `return _wrap_raw_ops_alias(alias)` instead